src/config.js

```diff
@@ -40,6 +40,10 @@ module.exports = {
 
   typeFormApiKey: process.env.TYPEFORM_API_KEY,
 
+  fbPageId: process.env.FACEBOOK_PAGE_ID,
```

Comment:
put the fb page id directly here, it's public information, and it's the same for dev/stage/prod